docs/quick-start.rst

```diff
@@ -62,6 +62,11 @@ If you haven't configured the CRUD plugin already, add the following lines to yo
          */
         public function beforeRender(\Cake\Event\Event $event)
         {
+            // For CakePHP 3.6+
```

Comment:
View::getClassName()/setClasName() are available since 3.4, so might as well mention `3.4+` in the comment.